src/global/interfaces/link-button-properties.ts

```diff
@@ -76,6 +76,27 @@ export interface ButtonProperties<T = any> extends AccessibilityProperties {
   emitButtonClick: (() => void) | undefined;
```

Comment:
Excuse me for coming late with one last comment. I was wondering if we could update the return type of this method, since otherwise if I needed to access the event in my component I could not.
```suggestion
  emitButtonClick: ((event: MouseEvent) => void) | undefined;
```

Reply:
Done!

Reply:
It's not always a MouseEvent. Do you really need this specific MouseEvent, or would it be ok to just go with Event?

Reply:
Just `Event` should be fine 👍